components/jupyter-web-app/rok/kubeflow/rokui/routes.py

```diff
@@ -46,15 +48,8 @@ def post_notebook_route():
 
   # Workspacae Volume
   if body["ws_type"] == "New":
-    pvc = create_pvc_template()
-    pvc['metadata']['name'] = body['ws_name']
-    pvc['metadata']['namespace'] = body['ns']
-    pvc['spec']['accessModes'].append(body['ws_access_modes'])
-    pvc['spec']['resources']['requests']['storage'] = \
-        body['ws_size'] + 'Gi'
-
     try:
-      create_pvc(pvc)
+      create_workspace_pvc(body)
```

Comment:
The code here looks very similar to components/jupyter-web-app/default/kubeflow/jupyter/server.py. Again, I wouldn't want to request any significant changes with so little context but it does seem like it would advantageous to factor out this code into a single shared module.